btrack/_localization.py

```diff
@@ -93,7 +93,7 @@ def _is_unique(x: np.ndarray) -> bool:
     else:
         # check to see whether the segmentation is unique
         if not _is_unique(segmentation):
-            labeled = label(segmentation)
+            labeled = label(segmentation, connectivity=1)
```

Comment:
Given that we use this function twice in the code, and it's parameterised - I would pull this out to a separate function. Something like `label_segmentation`